### ISSUE TITLE
Training hyperparameters

### ISSUE THREAD

**Issue by wojtekkutak** [Kaszkietio/CINIC10_CNN_Classifier#6]
Tune hyperparameters related to training (must be 2), for example: learning rate, exponential decay, momentum etc.

**Comment by wojtekkutak** [comment]
Done, results in dev branch.
